datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/{check_name}/datadog_checks/{check_name}/check.py

```diff
@@ -7,5 +7,28 @@
 class {check_class}(AgentCheck):
     def check(self, _):
         # type: (Any) -> None
-        # Use self.instance to read the check configuration
+        # The following are useful bits of code to help new users started.
+
+        # This is how you make HTTP Requests.
+        # More info at https://datadoghq.dev/integrations-core/base/http/
+        # self.http.get("<url>")
+
+        # This is how you submit metrics
+        # self.gauge("test", 1.23, tags=['foo:bar'])
+
+        # There are different types of metrics that you can submit (events, count).
+        # More info at https://datadoghq.dev/integrations-core/base/api/#datadog_checks.base.checks.base.AgentCheck
+        # self.count("count", 1.0)
```

Comment:
we don't submit counts anymore, use `monotonic_count`. Also perhaps we can just combine self.gauge and this step since users should use the link to discover the other metric types